nodeup/pkg/model/containerd.go

```diff
@@ -456,5 +463,35 @@ func (b *ContainerdBuilder) buildContainerdConfig() string {
 		// https://github.com/containerd/containerd/blob/master/docs/cri/config.md#cni-config-template
 		config.SetPath([]string{"plugins", "io.containerd.grpc.v1.cri", "cni", "conf_template"}, "/etc/containerd/config-cni.template")
 	}
-	return config.String()
+
+	if b.InstallNvidiaRuntime() {
+		if err := appendGPURuntimeContainerdConfig(config); err != nil {
+			return "", err
+		}
+	}
+	return config.String(), nil
+}
+
+func appendGPURuntimeContainerdConfig(config *toml.Tree) error {
```

Comment:
It is Nvidia specific and already in the containerd builder.
```suggestion
func appendNvidiaGPURuntimeConfig(config *toml.Tree) error {
```